app/lists.rb

```diff
@@ -0,0 +1,73 @@
+module Listing
+	extend Discordrb::Commands::CommandContainer
+	
+	wing_list = Array.new
+	#The array that is wing_list is created.
+	
+#  def self.find_role(event, role)
+#    has_role = false
+#    event.server.roles.each do |server_role|
+#      next unless %w(botadmin wingcaptain).any?(server_role.name)
+#      has_role = true
+#    end
+#    return role
+#  end
+
+  def self.role?(event, role)
+    has_role = false
+    event.server.roles.each do |server_role|
+      next unless server_role.name == role
+      has_role = true
+      break
+    end
+    return has_role
+  end
+	
+  command :winglist, description: "Lists those who are requesting wing." do |event|
+  #Simple, return the array's contents. Also, admin lock this.
+    break unless event.user.role?(find_role(event, 'wingcaptain'))
+    #Check to see if they have the "wingcaptain role"
+    event << wing_list.take(wing_list.length)
+	#Why do I have the feeling this will barf data unintelligibly?
+  end
+  
+  command :wingremove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event, target|
+  #Remove someone who is on the wing list. Defo admin lock.
+    break unless event.user.role?(find_role(event, 'wingcaptain'))
+	#Check to see if they have the "wingcaptain role". I am aware this is a double comment.
+	wing_list.delete(target)
+	event.respond "Removed" + target + "from the queue."
+  end
+  
+  command :wingnuke, description: "Removes everything from the wing list. Useful for after a meetup." do |event|
+    break unless event.user.role?(find_role(event, 'wingcaptain') && find_role(event, 'botadmin'))
+	#Check to see if they have the "wingcaptain" role AND "botadmin" role. Probably won't work.
+	wing_list.clear
+	#will this actually work? I think no, but i can't think of a better way.
+	event.respond << "As requested, the list has been nuked."
+  end
+
+  command :wingme, description: "Need a wing? &WingMe" do |event|
```

Comment:
Sorry, this description's been bothering me. Could you make it something like 'Add yourself to the wing queue'?